src/clients.js

```diff
@@ -1,33 +1,34 @@
 /* @flow */
-import AppRegistryClient from '@vtex/api/lib/AppRegistryClient'
-import AppEngineClient from '@vtex/api/lib/AppEngineClient'
-import WorkspacesClient from '@vtex/api/lib/WorkspacesClient'
-import RouterClient from '@vtex/api/lib/RouterClient'
-import VTEXIDClient from '@vtex/api/lib/VTEXIDClient'
+import type {RegistryInstance, AppsInstance, WorkspacesInstance, RouterInstance} from '@vtex/api'
+import {Registry, Apps, Workspaces, Router, ID} from '@vtex/api'
 import endpoint from './endpoint'
 import {version} from '../package.json'
+import {getAccount, getWorkspace} from './conf'
 
 const options = {
+  account: getAccount(),
+  workspace: getWorkspace(),
   authToken: 'abc123',
   userAgent: `Toolbelt/${version}`,
+  region: 'aws-us-east-1',
 }
 
-export function registry (): AppRegistryClient {
-  return new AppRegistryClient(endpoint('registry'), options)
+export function registry (): RegistryInstance {
+  return Registry({...options, endpoint: endpoint('registry')})
```

Comment:
If the options are fixed, I think we should export a simple object. This way we don't recreate clients each time.